### ISSUE TITLE
File manager to start the game

### ISSUE THREAD

**Issue by Chenwq297** [nikita36078/J2ME-Loader#1000]
If j2me already has the jar installed, using the file manager to launch the jar to call j2me will display the installation page. Is there a way to directly call it to start the game?

**Comment by Chenwq297** [comment]
j2me已安装该jar的情况下使用文件管理器启动jar来调用 j2me 会显示安装页面，有没有办法直接调用来启动游戏？

**Comment by woesss** [comment]
If the game is already installed, then the installer window contains a button to run without reinstalling.  
This is a minor hurdle compared to how many taps it takes in the game itself to get to the gameplay.

**Comment by Chenwq297** [comment]
> 

Yes, I just want to know if there is a way to start it directly without clicking the button again
对，就是想知道有没有办法直接启动，而不用再点击一次按钮

**Comment by woesss** [comment]
There is no such way. Emulator can't know when you want to run and when to reinstall

**Comment by Noitarud** [comment]
I presume this is for compatibility with Ghost commander's "add shortcut to home screen" (or ZH equivilent) - perhaps J2MELoader could have its own version of this function and have this app place its own icons on the home screen#. If the Java app does not supply an icon for itself you could enable the user to draw one*.

*Contacts app allows taking of picture (camera)for contact, however dialogue comes up saying "capture image with [app]" and here I can select enabled draw programs (such as simple draw pro) besides camera app. #the contacts app can place contacts on home screen too.

**Comment by Chenwq297** [comment]
> 

I want to use pegasus-frontend to start J2me
https://github.com/mmatyas/pegasus-frontend

**Comment by SergioFLS** [comment]
and i would love enhanced integration with Ox Shell! https://github.com/oxters168/OxShell

**Comment by woesss** [comment]
@SergioFLS, After all, this is a shell for Android, and not a front-end for emulators - what kind of integration is possible there?
